content/articles/index.md

```diff
@@ -0,0 +1,71 @@
+### Getting Started With Ray Tracing Algorithm in Machine Learning
+
+### Prerequisites
+- The reader should know what's an algorithm and how it's used in programming.
+
+- The reader should be familiar with data structures and neural networks.
+
+
+### Introduction
+When playing video games, have you ever wondered how or rather what makes the light in the game look similar to the one in real life? This is because of the *ray tracing technique*, with this process, the game designers can make visible light beam jump into objects, produce real shadows, and create life-like reflections.
+
+Ray tracing has been of great impact in the world today, modifying and innovating various sectors of the tech world. In this article, we'll get to know about ray tracing algorithm in machine learning.
+
+
+### Tables of contents
+- [What is Ray Tracing Algorithm](#what-is-ray-tracing-algorithm)
+- [History of Ray Tracing](#history-of-ray-tracing)
+- [Types of Ray Tracing Algorithm](#types-of-ray-tracing-algorithm)
+  - [Hybrid Ray Tracing Algorithm](#hybrid-ray-tracing)
+  - [Forward Ray Tracing Algorithm](#forward-ray-tracing)
+  - [Backward Ray Tracing Algorithm](#backward-ray-tracing) 
+ - [Conclusion](#conclusion)
+
+
+### What is Ray Tracing Algorithm and how does it work?

```

Comment:
Split this section. Create a section for 'What is Ray tracing algorithm' and another one for 'how ray tracing algorithm works'. Each of these sections should be organized in reasonable paragraphs.